packages/expo-cli/bin/expo.js

```diff
@@ -13,9 +13,8 @@ const minor = parseInt(match[2], 10);
 
 const supportedVersions =
   'expo-cli supports following Node.js versions:\n' +
-  '* >=12.13.0 <13.0.0 (Maintenance LTS)\n' +
-  '* >=14.0.0 <15.0.0 (Active LTS)\n' +
-  '* >=15.0.0 <17.0.0 (Current Release)\n';
+  '* >=12.13.0 <15.0.0 (Maintenance LTS)\n' +
```

Comment:
```suggestion
  '* >=12.13.0 <16.0.0 (Maintenance LTS)\n' +
```

Reply:
Node 15 is considered 'current', not 'Maintenance LTS', that's why I excluded 15, and I didn't want to list it separately because we advise LTS version.